MBBSEmu/Module/MsgFile.cs

```diff
@@ -42,145 +42,179 @@ public MsgFile(IFileUtility fileUtility, string modulePath, string msgName)
             BuildMCV();
         }
 
+        private enum MsgParseState {
+            NEWLINE,
+            IDENTIFIER,
+            SPACE,
+            BRACKET,
+            JUNK
+        };
+
+        private static bool IsIdentifier(char c) =>
+            Char.IsDigit(c) || (c >= 'A' && c <= 'Z');
```

Comment:
`char`

Reply:
isn't this the same thing being a static method?

Reply:
it's more just a style thing to reference the actual type `char` vs. the alias `Char`